website/project/model.py

```diff
@@ -3918,13 +3927,19 @@ def update_metadata(self, metadata):
         self.registration_metadata.update(metadata)
         return changes
 
+    def add_status_log(self, action):
+        self.status_logs.update({
+            datetime.datetime.utcnow().strftime('%Y-%m-%d %H:%M'): action
```

Comment:
This is a really simple log for David just to be able to see how many stages one of these went through. It would certainly be possible to make a new model of a stripped down log, but I thought that it would be simpler to have a list, however a list really isn't supported in modm beyond using other models.